experimental/dds/tree2/src/test/shared-tree/fuzz/targetedFuzz.spec.ts

```diff
@@ -39,9 +39,7 @@ interface AbortFuzzTestState extends FuzzTestState {
 describe("Fuzz - Targeted", () => {
 	const opsPerRun = 20;
 	const runsPerBatch = 20;
-	const editGeneratorOpWeights: Partial<EditGeneratorOpWeights> = {
-		setPayload: 1,
-	};
+	const editGeneratorOpWeights: Partial<EditGeneratorOpWeights> = {};
```

Comment:
@daesun-park Are there other weights we would want to set instead?

Reply:
You could try putting `insert: 1` and see if that passes. If not, just leave it empty.